packages/salesforcedx-vscode-soql/package.json

```diff
@@ -69,7 +69,9 @@
     "papaparse": "^5.3.0",
     "sinon": "7.5.0",
     "typescript": "3.8.3",
-    "vscode-languageclient": "6.1.3"
+    "vscode-languageclient": "6.1.3",
+    "webpack": "4.30.0",
```

Comment:
without this, we can't actually run `npm run compile` from within the `packages/salesforcedx-vscode-soql` directory.  which is a huge time saver.